sql/src/test/java/org/apache/druid/sql/calcite/CalciteArraysQueryTest.java

```diff
@@ -3052,6 +3066,7 @@ public void testUnnestThriceWithFiltersOnDimAndUnnestColumnsORCombinations()
   {
     cannotVectorize();
     skipVectorize();
+    notMsqCompatible();
```

Comment:
`lotsofcolumns` dataset is not loaded into MSQ tests now, once we load it we can remove this

Reply:
Lets add the above comment regarding the datasource `lotsofcolumns`in the UT itself so that we know how to enable it.

Reply:
Adding this should be easy: https://github.com/apache/druid/blob/39d95955f5508143b4fd2352baa3fe60a4920357/extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/CalciteMSQTestsHelper.java#L182 You can make the changes here. We should enable this in the PR itself unless it requires some other refactor.